### DISCUSSION TITLE
ISSUE: mising outline in dictionary -  malicious. PHA/HREURB/KWRUS

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#22] [General]
In exercises for  CH 15 there is an entry malicious	PHA/HREURB/KWRUS
this outline is not in the dictionaries, but there are different outlines without KWR.

Should this word be removed from the exercises, or the outline corrected, or this suggested outline added to some dictionary?

for me "ma li shus" (present in the user dict)  - makes more sense than "ma - lish - yus", but for me English is not native.

**Comment by aerickt** [comment]
Thanks, removed in https://github.com/aerickt/lapwing-for-beginners/commit/5a3a99eb014698fba6f5fbb97a0e45a893b11eb9.
